README.md

```diff
@@ -45,9 +43,7 @@ function Component(props) {
 export default Component;
 ```
 
-What I like about this pattern is that it's clear that the styles you're writing are for a component and you can even have the integrate some styling APIs right into the component's API (i.e. its props).
-
-For example, in Material UI, a parent component can override `title` styles like so:
+This pattern rocks because it creates styles on a component level and it's simple for a parent component to override child styles. For example, in Material UI, a parent component can override `title` styles like so:
```

Comment:
too strong lol

```suggestion
This pattern is great because it creates styles on a component level and it's simple for a parent component to override child styles. For example, in Material UI, a parent component can override `title` styles like so:
```

Reply:
👍